meeting/models.py

```diff
@@ -45,8 +45,8 @@ def __str__(self):
         return "meeting " + str(self.meeting) + " from " + str(self.start_time) + " to " + str(self.end_time)
 
 class Meeting(models.Model):
-    start_date = models.DateField(default = '0000/00/00')
-    end_date = models.DateField(default = '0000/00/00')
+    start_date = models.DateField(default = '1970-01-01')
+    end_date = models.DateField(default = '1970-01-01')

```

Comment:
Migrations didn't like the YYYY/MM/DD format (didn't like the slashes) and it didn't like a zero-date, so I set it to the UNIX epoch.

Reply:
That'll work. Thank you for fixing it.

Reply:
absolutely.